usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -647,14 +647,12 @@ def load_file_d1(submission_attributes, procurement_data, db_cursor):
         legal_entity_location, created = get_or_create_location(legal_entity_location_field_map, row, legal_entity_location_value_map)
 
         # Create the legal entity if it doesn't exist
-        try:
-            legal_entity = LegalEntity.objects.get(recipient_unique_id=row['awardee_or_recipient_uniqu'])
-        except ObjectDoesNotExist:
+        legal_entity, created = LegalEntity.get_or_create_by_duns(duns=row['awardee_or_recipient_uniqu'])
+        if created:
```

Comment:
Maybe not something we need to address right now, but do you think we'll ever have to update attributes on an existing legal entity based on the incoming data? I suspect not, since our incoming records aren't the canonical DUNS source, but seems worth asking.